src/Aspire.Hosting/ApplicationModel/AspireStore.cs

```diff
@@ -18,6 +18,11 @@ public AspireStore(string basePath)
     {
         ArgumentNullException.ThrowIfNull(basePath);
 
+        if (!Path.IsPathRooted(basePath))
```

Comment:
Can we ensure it is rooted when we put it in the AssemblyMetadataAttribute?

https://github.com/dotnet/aspire/blob/0ab5904e229f339035caf9d3e45f590badc7321b/src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.in.targets#L245-L252

Reply:
Right. I realized when the poster explained their value was `obj/`. I assume in the aspire solution we rewrite it for the artifacts folder. But the default for external project is `obj/` as per documentation.

Does the change seem right? I tested it with different values.